README.md

```diff
@@ -12,7 +12,8 @@ Role Variables
 --------------
 
 Optional variables:
-- `java_jre_versions`: A list of JRE versions to install, default `[1.8.0]`
+- `java_jre_versions`: A list of Java versions to install, default `["8"]`,
```

Comment:
Could do

Reply:
Done, in practice the variables behave the same since the package names are also inconsistent and attempting to use different logic for each variable would make it even more complicated. On option would be to add an org level variable like `ome_error_deprecation` to make it easy to find deprecated variables which are still handled.